Resources/Prototypes/Entities/Objects/Weapons/Melee/e_sword.yml

```diff
@@ -387,12 +387,65 @@
 # Borgs
 - type: entity
   suffix: One-Handed, For Borgs
-  parent: EnergySwordDouble
+  parent: EnergySword #To avoid the evil wielded component
   id: CyborgEnergySwordDouble # why is this invalid if ID is BorgEnergySwordDouble
+  name: double-bladed energy sword
   description: Syndicate Command Interns thought that having one blade on the energy sword was not enough. Specially designed for syndicate cyborgs.
   components: # could add energy-draining like the L6C
-  - type: Wieldable
-    freeHandsRequired: 0 # because borg has no off-hand to wield with.  Without this, it will be unable to activate the esword
+  - type: ItemToggle
```

Comment:
this is a goofy amount of yaml duplication. you can use composition to reduce like 99% of this